### DISCUSSION TITLE
Pytest on MU editor

### DISCUSSION THREAD

**Discussion by BarboraKa** [mu-editor/mu#2401] [Q&A]
Hi,

how do I run a pytest on MU editor? I downloaded it as a 3rd party package but I just can't figure out where I should write what line of code to actually run the test. I am doing CS50 Introduction to programming and the way they do it there (they use VS code) through terminal just doesn't work for me. So how exactly do I run it? I tried writing  pytest.test_calculator.py into the code or into my REPL and also tried other ways but nothing works.

Thank you!

**Comment by haikusw** [comment]
You can either:
a) change to the mu/tests directory and run pytest there
b) pass a path to pytest to tell it where the tests are, e.g., `pytest ~/Documents/mu/tests/`
